scripts.js

```diff
@@ -1,3 +1,9 @@
+
+$(function(){
+    
```

Comment:
ok, but now this function is empty :) all your code should be inside, only then your code will run after DOM is loaded